gcloud/datastore/client.py

```diff
@@ -51,53 +130,168 @@ def __init__(self, dataset_id=None, namespace=None, connection=None):
         self.namespace = namespace
 
     def get(self, key, missing=None, deferred=None):
-        """Proxy to :func:`gcloud.datastore.api.get`.
+        """Retrieve an entity from a single key (if it exists).
 
-        Passes our ``dataset_id``.
+        .. note::
+
+           This is just a thin wrapper over :method:`get_multi`.
+           The backend API does not make a distinction between a single key or
+           multiple keys in a lookup request.
```

Comment:
I think the linter will complain about undocumented args here, but I will too.

From the old method

```python
    :type key: :class:`gcloud.datastore.key.Key`
    :param key: The key to be retrieved from the datastore.

    :type missing: an empty list or None.
    :param missing: If a list is passed, the key-only entities returned
                    by the backend as "missing" will be copied into it.
                    Use only as a keyword param.

    :type deferred: an empty list or None.
    :param deferred: If a list is passed, the keys returned
                     by the backend as "deferred" will be copied into it.
                     Use only as a keyword param.
```